src/lib/components/ui/menu/menu-title.svelte

```diff
@@ -20,17 +23,21 @@
 		element = 'span',
 		disabled,
 		content,
+		hover,
+		active,
+		focus,
 		children,
 		class: className = '',
 		...restProps
 	}: Props = $props()
 	let labelFor = Math.random().toString(16)
-	const store = withMenuStore(dimension, disabled)
+	const store = withMenuStore(dimension, open, disabled)
 	setContext('menu-store', store)
 
 	$effect(() => {
 		store.size = dimension
 		store.disabled = disabled
+		store.open = open
 	})
```

Comment:
Not for my investigation: if any value changes, does this re-render the component 3 times or just 1 time? Can this stay as single effect or should we split it into 3?